packages/terra-site/src/examples/card/CardPaddingVertical.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import Card from 'terra-card';
+
+const CardPaddingVertical = () => (
+  <div>
+    <Card>
+      <Card.Padding hasPaddingVertical>Hello World!!</Card.Padding>
```

Comment:
up to you, i think its mainly to show the HR can reach the edge, whilst the top and lower regions have a re-usable way to declare their padding from the edges. This does bring up a good point though that you couldn't use the padding component to declare padding between just the Top of the card and the Bottom. The vertical padding would put padding between each region and the HR...